src/pagination.rs

```diff
@@ -271,50 +271,73 @@ impl<'a> Category<'a> {
 }
 
 // only used in parse_categories_list, to transform liquid::value::Array to type with Ord trait
-fn liquid_array_to_string(arr: &liquid::value::Array) -> String {
-    let mut s: String = arr
-        .iter()
-        .map(|i| {
-            let mut s = i.to_str().to_string();
-            s.push('/');
-            s
-        })
-        .collect();
-    s.pop(); // remove last '/'
-    s
+// fn liquid_array_to_string(arr: &liquid::value::Array) -> String {
+//     let mut s: String = arr
+//         .iter()
+//         .map(|i| {
+//             let mut s = i.to_str().to_string();
+//             s.push('/');
+//             s
+//         })
+//         .collect();
+//     s.pop(); // remove last '/'
+//     s
+// }
+
+fn compare_category_path(p1: &liquid::value::Array, p2: &liquid::value::Array) -> Ordering {
+    p1.iter()
+        .partial_cmp(p2.iter())
```

Comment:
Oh cool!

Reply:
Looking cool, but I don't know if it's the correct way to write the comparison function because of the issue above ^^'